src/psbt.cpp

```diff
@@ -325,3 +330,139 @@ TransactionError CombinePSBTs(PartiallySignedTransaction& out, const std::vector
 
     return TransactionError::OK;
 }
+
+std::string PSBTRoleName(PSBTRole role) {
+    switch (role) {
+        case PSBTRole::UPDATER:   return "updater";
```

Comment:
I think we align `case` with `switch`. nit, I'd ignore `return`s alignment.

Reply:
Done.